tools/lkl/lib/virtio_net_dpdk.c

```diff
@@ -345,6 +385,12 @@ struct lkl_netdev *lkl_netdev_dpdk_create(const char *ifparams, int offload,
 	char poolname[RTE_MEMZONE_NAMESIZE];
 	char *debug = getenv("LKL_HIJACK_DEBUG");
 	int lkl_debug = 0;
+	int max_packet_sz = MAX_PACKET_SZ;
+	int mbuf_num = MBUF_NUM;
+	int mbuf_size = 0;
+	int ifparams_len = 0;
+	char *ifparams_local = NULL;
```

Comment:
times fly: since https://github.com/lkl/linux/pull/385 introduces JSON configuration interface, it would be nicer that this configuration can be done via this new way: you can have your private parameters as a json key to configure internal behavior of your code.

If you want to keep this parameter hidden, you can just implement it. if you intended to be a public one, then you can document in lkl.txt.